src/sync/installation.ts

```diff
@@ -375,6 +383,9 @@ export const processInstallation =
 					task,
 					repositoryId
 				);
+
+				statsd.increment(metricTaskStatus.complete, [`type: ${nextTask.task}`]);
```

Comment:
What is `nextTask.task`?  Is it type of data this task is synching? (Just want to make sure that variations of this value are bounded)

Reply:
The variable `nextTask` contains some info about the current task (the naming is terrible). Yeah, `nextTask.task` is either `commit`, `pull` or `branch` like the keys in the `tasks` variable at the top of the file. 

This code is terrible, we need to refactor it.